prime-router/src/main/kotlin/metadata/Mappers.kt

```diff
@@ -529,6 +529,48 @@ class LookupSenderValuesetsMapper : Mapper {
     }
 }
 
+/**
+ * The lookupSenderAutomationValuesetsMapper is used to lookup values from the
+ *      "sender_automation_value_set_row" table/csv
+ * The args for the mapper are:
+ *      args[0] --> senderAutomationValueSetId = the id of the sender automation value set from
+ *          sender_automation_value_set primary lookup field
+ * The mapper uses the above arguments + the question's answer to retrieve a row from the table
+ */
+class LookupSenderAutomationValuesetsMapper : Mapper {
+    override val name = "lookupSenderAutomationValuesetsMapper "
```

Comment:
Also wondering this. Looking at both, it seems `LookupMapper` would work.

Reply:
@kevinhaube i think you might be right, since we're only looking up w/one key. but I would put forth here the idea to stay with the .filter() way anyway because 1. I remember the schema setup for that way being a lot more complicated and 2. on a future iteration we might be adding more fields to look up against to account for sender-specific lookups, versioning, etc. if we stayed with this method, it would be much simpler to extend IMO

Reply:
@carlosfelix2 the mapper on line 497, LookupSenderValuesetsMapper() was a mapper that I implemented back in December as part of a AOE fuzzy lookup mapper. I don't believe anyone is using it (for whatever reason), so we were thinking about either replacing it entirely with _this_ new mapper OR perhaps marking it as depreciated somehow (do we do this in Kotlin here currently?) Replacing it entirely would solve the long name problem, but do we really want to remove "Mapper" from the name, in any event? Every other mapper in Mappers.kt follows that convention.